grakn-engine/src/main/java/ai/grakn/engine/tasks/manager/singlequeue/SingleQueueTaskManager.java

```diff
@@ -135,7 +136,8 @@ public SingleQueueTaskManager(EngineID engineId) throws Exception {
 
         EngineCacheProvider.init(EngineCacheDistributed.init(zookeeper));
 
-        LockProvider.add(POST_PROCESSING_LOCK, new ZookeeperLock(zookeeper, LOCK));
+        LockProvider.add(PostProcessingTask.LOCK_KEY, new ZookeeperLock(zookeeper, ZookeeperPaths.LOCK));
```

Comment:
Because these two locks are on the same path (second argument of creating `ZookeeperLock`), they will lock with one another. They should be on two paths.